workflows/methylation/methylation-preprocess.wdl

```diff
@@ -0,0 +1,209 @@
+version 1.1
+
+workflow methylation_preprocess {
+    meta {
+        description: "Process raw IDAT files from the Illumina methylation array for a single sample"
+        outputs: {
+            beta_swan_norm_unfiltered: "Normalized beta values for all probes",
+            beta_swan_norm_unfiltered_genomic: "Normalized beta values for all probes that map to the genome",
+            mset: "MethylSet object",
+            rgset: "RGSet object",
+            rset: "RatioSet object",
+            annotation: "Annotation object",
+            beta: "Beta values",
+            cn_values: "Copy number values",
+            m_values: "M values",
+            pheno: "Phenotype data",
+            pheno_data: "Phenotype data",
+            probe_names: "Probe names",
+            sample_names: "Sample names",
```

Comment:
Some of these need to be removed. Charlie's original script output a lot of intermediate files, which we don't really need.

Reply:
Should we create an `IntermediateFiles` struct like we have for QC? IDK if that would be useful, just an idea. Is there a context where a user would want the intermediate files? Maybe even just for debugging?

Reply:
I'll give it some thought, but I don't think there is any value to most of these files. Certainly not `sample_names` because that will have one entry with the name of the current sample. I can see outputting the M value and CN value as those could be useful.